src/me/mrCookieSlime/CSCoreLibPlugin/general/Player/PlayerStats.java

```diff
@@ -49,7 +49,7 @@ public static PlayerStats getStats(Player p) {
 	public long getStatistic(PlayerStat stat) {
 		switch (stat) {
 			case PLAY_TIME_MS: {
-				return Long.valueOf(String.valueOf(Bukkit.getPlayer(uuid).getStatistic(Statistic.PLAY_ONE_TICK) * 50));
+				return Long.valueOf(String.valueOf(Bukkit.getPlayer(uuid).getStatistic(Statistic.PLAY_ONE_MINUTE) * 50));
```

Comment:
Minutes and Ticks are not the same, so the values wouldn't add up in the end.
Please adjust the math of this to reflect that.

Reply:
I will fix this

Reply:
Done